ContributedExamples/plot_mht.ipynb

```diff
@@ -4,27 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>I suggest adding a short description of what this notebook does at the top here. I'd suggest also including:</p><ol><li>This notebook calculates the model MHT from the MOM5 diagnostic <code>temp_yflux_adv_int_z</code> . So to use it this diagnostic needs to be available in the output.</li><li> Currently, only the total (all basins) MHT is calculated.</li><li>Includes comparison to a few observational products.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/227/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I think that is a good idea. To be honest, I am not familiar with the calculations that are done here, so I could not include more info, but this is great.